app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -152,5 +153,50 @@
         });
       }
     }
+
+    /*
+    * Check that the current item is infusable
+    * Only legendary or Exotic items are infusable.
+    */
+    vm.isInfusable = function isInfusable(item) {
+      // Infuse perk's id is 1270552711
+      return _.contains(item.talentPerks, 1270552711);
+    }
+
+    /*
+    * Show the icon for infusion, weapon or armor
+    */
+    vm.icon = function icon(item) {
+      var icon = "";
+      if (item.sort && item.sort === "Weapons") {
```

Comment:
Even for the user to know to open the move dialog and know the infuse icon is actually a button that does something. I'm sure they will catch on, but going in to it blind it mightt just look like another perk on the item (which you only see if you click the info icon)